typescript/simple-mercari-web/src/components/Listing.tsx

```diff
@@ -58,20 +77,23 @@ export const Listing = ({ onListingCompleted }: Prop) => {
             placeholder="name"
             onChange={onValueChange}
             required
+            value={values.name}
```

Comment:
Fixed a bug where the application's state was not reflected in the Form UI.
related : https://mercari.slack.com/archives/C04SV7GD5U1/p1740795241738709?thread_ts=1740784386.962209&cid=C04SV7GD5U1